### PR TITLE
Improve PR Issue Template info in order to get required data

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Motivation
Improve PR Issue Template info in order to get required data

## What
- Request the version of OCP and Ansible
- Add command to help the user obtained it.
 
## Why
- Issues can be faced in specific versions of OCP and/or Ansible

## Checklist:

- [X] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [x] Finished task
- [ ] TODO

## Additional Notes

screenshot 2018-11-19 at 22 07 14

### CODE REVIEW

**Comment by davidffrench** [comment]
@camilamacedo86 I don't believe GitHub renders comments when displaying the description. Could you please confirm.

**Comment by camilamacedo86** [comment]
@davidffrench It is for the issue template. It will appear as it is the text view. However, If the user did not remove the descriptions <-- --> it will not appear in the issue just the things that they write down.

File: .github/iSSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
-Describe the current/actual behaviour.
+<!-- Describe the current/actual behaviour. -->
 
 ## Environment
 
-* Operating system:
+* Operating system: (E.g RHEL 7.6 )
 * OpenShift version:
-* Project Version/Tag:
+<!-- Run the command `oc version` and add here the result. -->
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply .github/iSSUE_TEMPLATE.md]: 👍 I will fix before merging tks

**Comment by camilamacedo86** [comment]
@davidffrench @pb82 hi, could you please approve this one for we are able to move forward here?

**Comment by pb82** [comment]
@camilamacedo86 done

**Pull request merged**